### PR TITLE
simplify MSSQL images

### PR DESCRIPTION
**Pull request by stephane-airbyte**
remove MSSQL_2022_UBUNTU_2024. Rename into MSSQL_VERSION, get everything on latest

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
Current dependencies on/for this PR:
* master
  * **PR #32851** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #32876** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32876?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #32897** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32897?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #32898** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32898?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #32899** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32899?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #32900** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32900?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #32948** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32948?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #32951** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32951?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/32951?utm_source=stack-comment).

**Comment by stephane-airbyte** [comment]
Current dependencies on/for this PR:
* master
  * **PR #32851** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #32876** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32876?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #32897** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32897?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #32898** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32898?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #32899** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32899?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #32900** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32900?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #32948** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32948?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #32951** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32951?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/32951?utm_source=stack-comment).

File: airbyte-integrations/connectors/source-mssql/src/testFixtures/resources/testDockerFile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ARG BASE_IMAGE=mcr.microsoft.com/mssql/server:server:2022-latest
+FROM ${BASE_IMAGE}
```
Comment (postamar) [inline airbyte-integrations/connectors/source-mssql/src/testFixtures/resources/testDockerFile:2]: Did you commit this by mistake?

**Review by postamar — APPROVED** [review]

**Comment by stephane-airbyte** [comment]
### Merge activity

* **Dec 7, 7:42 PM**: [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/32951) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**